src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -125,6 +125,58 @@ const AccessibilityRequestDetailPage = () => {
 
   const userGroups = useSelector((state: AppState) => state.auth.groups);
   const isAccessibilityTeam = user.isAccessibilityTeam(userGroups, flags);
+  const hasDocuments = documents.length > 0;
```

Comment:
It's odd to me that the parent of the doc list is controlling the empty state. It feels like the doc list itself should control the empty/not empty state? That might mean moving the button into the list component, which would also make sense to me.

Reply:
If ya'll need a tie breaker, I'm happy with the way it is. I'm free to chat if there are Q's.

Reply:
I ended up leaving it as is.